lib/ansible/modules/network/ios/ios_banner.py

```diff
@@ -107,25 +106,23 @@ def map_obj_to_commands(updates, module):
         if want['text'] and (want['text'] != have.get('text')):
             banner_cmd = 'banner %s' % module.params['banner']
             banner_cmd += ' @\n'
-            banner_cmd += want['text'].strip()
+            banner_cmd += want['text'].strip('\n')
             banner_cmd += '\n@'
             commands.append(banner_cmd)
 
     return commands
 
 
 def map_config_to_obj(module):
-    rc, out, err = exec_command(module, 'show banner %s' % module.params['banner'])
-    if rc == 0:
-        output = out
-    else:
-        rc, out, err = exec_command(module,
-                                    'show running-config | begin banner %s'
-                                    % module.params['banner'])
-        if out:
-            output = re.search(r'\^C(.*?)\^C', out, re.S).group(1).strip()
```

Comment:
@rohitthakur2590 can u write a comment for the change.

Reply:
so with exec_command we were getting output with stripped initial spaces or  newline ,  so I have refactored it to get the exact config without any formatting, and then carving out the banner from it.

Reply:
@rohitthakur2590 can u write the same as code comment

Reply:
Done.